recipes/atlasgo/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.21.1" %}
+
+package:
+  name: atlasgo
+  version: {{ version }}
+
+source:
+  url: https://github.com/ariga/atlas/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eb35bf79613d1bba10df482cf688a0c5a139e8834f4ab65747cd2b63cbe610ae
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [unix]
+    - {{ stdlib('c') }}    # [osx]
```

Comment:
```suggestion
    - {{ stdlib('c') }}    # [unix]
```

Reply:
See the second to last commit - this actually didn't work since the environment could not be solved with a Go >=1.21 compiler 🧐

Reply:
I cannot see the error. What was the problem?

Reply:
Let me commit your suggestion then we should see it again :eyes:

Reply:
@xhochy, this should be the relevant part of the log:

```
Could not solve for environment specs
The following packages are incompatible
├─ go-cgo_linux-64 >=1.21  is installable and it requires
│  ├─ _go_select ==2.3.0 cgo, which can be installed;
│  └─ go [1.21.1.* |1.21.3.* |...|1.22.2.* ] with the potential options
│     ├─ go [1.21.1|1.21.3|...|1.22.2] would require
│     │  └─ sysroot_linux-64 2.17.* , which can be installed;
│     └─ go [1.21.1|1.21.3|...|1.22.2] would require
│        └─ _go_select ==2.2.0 nocgo, which conflicts with any installable versions previously reported;
└─ sysroot_linux-64 2.12.*  is not installable because it conflicts with any installable versions previously reported.
```

Reply:
@xhochy guess I fixed it via the build config. I only discovered `sysroot_linux` in the stdlib migration news now :eyes: